src/button-groups/button-group.ts

```diff
@@ -0,0 +1,41 @@
+import { Component, Input, Output, EventEmitter, ChangeDetectionStrategy, ElementRef, Renderer } from 'angular2/core';
+import { BehaviorSubject } from 'rxjs/subject/BehaviorSubject';
+
+@Component({
+  selector: 'ngl-button-group',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  template: `
+    <div class="slds-button-group" role="group">
+      <ng-content></ng-content>
+    </div>
+  `,
+})
+export class NglButtonGroup {
+  @Output() selectedChange = new EventEmitter(false);
+
+  @Input() set selected(_selected: string) {
+    this._selected = _selected;
+    this.values.next(this._selected);
+  };
+
+  get selected() {
+    return this._selected;
+  }
+
+  values = new BehaviorSubject(null);
+  private _selected: string;
+
+  constructor(public element: ElementRef, public renderer: Renderer) { }
```

Comment:
I'd suggest turning `element` and `renderer` to private members for better encapsulation.